inst/include/date.h

```diff
@@ -0,0 +1,4528 @@
+#ifndef DATE_H
+#define DATE_H
+
+// The MIT License (MIT)
+//
+// Copyright (c) 2015, 2016 Howard Hinnant
```

Comment:
Hahhahhaa. I was wondering when I would have to give in an include his. He is unavoidable :)  And good, this is now in C++20 as well.   

(Should we farm it out to another package though to be more widely useable?  Just a thought....)